lib/nerves_network/dhcpv6_manager.ex

```diff
@@ -176,10 +176,17 @@ defmodule Nerves.Network.DHCPv6Manager do
   defp consume(:dhcpv6, {:deconfig, _info}, state), do: state
 
   defp consume(:dhcpv6, {:bound, info}, state) do
-    Logger.debug fn -> "#{__MODULE__}: consume :bound info: #{inspect info}" end
+    Logger.debug fn -> "#{__MODULE__}: consume :dhcp6 :bound info: #{inspect info}" end
     state
       |> configure(info)
-      |> goto_context(:up)
+      |> goto_context(:dhcpv6)
+  end
+
+  defp consume(:dhcpv6, {:renew, info}, state) do
```

Comment:
This function and the one above look the same, just the :bound and :renew are different in the function head.

```suggestion
  defp consume(:dhcpv6, {action, info}, state) when action in [:renew, :bound] do
```

Reply:
I was thinking about it. Duplication of the code in the case of a state machine I deem is preferable. One can modify the 'DO' of the states independently when the need arises. With the proposed construction the future 'fixer' is unlikely to split the two - it is more of psychological approach rather than compactness oriented.

Reply:
I have no objection to that.